test/e2e/test-plugins/custom-cpts.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Plugin Name: Gutenberg Test Custom CPTS
+ * Plugin URI: https://github.com/WordPress/gutenberg
+ * Author: Gutenberg Team
+ *
+ * @package gutenberg-test-custom-cpts
+ */
+
+add_action( 'init', function() {
```

Comment:
Looks like there are a bunch of PHPCS errors here. Are these *spaces* you're using?! 😉